substrate-processor/src/batch.ts

```diff
@@ -14,6 +14,7 @@ export interface DataHandlers {
      * Mapping of type `trigger event` -> `extrinsic` -> `extrinsic handler list`
      */
     extrinsics: Record<QualifiedName, Record<QualifiedName, ExtrinsicHandler[]>>
+    evmLogs: Record<QualifiedName, EvmLogHandler[]>
```

Comment:
`QualifiedName` has a meaning of `${section}.${item}`. Contract address definitely falls outside of this category, please create a separate alias for that.